src/app/shared/services/env/env.service.ts

```diff
@@ -1,17 +1,24 @@
-export class EnvService {
-
-  theme = 'default';
+import { Injectable } from "@angular/core";
+import { environment } from "../../../../environments/environment";
 
-  rpcUrl = 'https://volta-rpc-vkn5r5zx4ke71f9hcu0c.energyweb.org/';
-  chainId = 73799;
-
-  cacheServerUrl = 'https://identitycache-dev.energyweb.org/v1';
-  natsServerUrl = 'https://identityevents-dev.energyweb.org/';
-  kmsServerUrl = undefined;
-  stakingPoolFactoryAddress;
-  claimManagerAddress;
-  showAzureLoginOption = true;
-  isWalletConnectEnabled: boolean;
-  checkStakingVerification: boolean;
-  patronRole: string;
+@Injectable({
+  providedIn: "root",
+})
+export class EnvService {
+  theme = environment.theme;
+  rpcUrl = environment.rpcUrl;
+  chainId = environment.chainId;
+  cacheServerUrl = environment.cacheServerUrl;
+  natsServerUrl = environment.natsServerUrl;
+  kmsServerUrl = environment.kmsServerUrl;
+  stakingPoolFactoryAddress = environment.stakingPoolFactoryAddress;
+  checkStakingVerification = environment.checkStakingVerification;
+  showAzureLoginOption = environment.showAzureLoginOption;
+  isWalletConnectEnabled = environment.isWalletConnectEnabled;
+  patronRole = environment.patronRole;
+  claimManagerAddress = environment.claimManagerAddress;
+  networkName = environment.networkName;
+  currencyName = environment.currencyName;
+  currencySymbol = environment.currencySymbol;
+  blockExlorerUrl = environment.blockExlorerUrl;
```

Comment:
just a typo.
```suggestion
  blockExplorer = environment.blockExplorerUrl;
```

Reply:
isn't it  `blockExlorerUrl = environment.blockExlorerUrl;` ?

Reply:
maybe :D do not change it, just asked :D

Reply:
Yeah, I am using it in metamask import like that